SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -216,6 +216,18 @@ void USpatialGameInstance::HandleOnConnectionFailed(const FString& Reason)
 	OnConnectionFailed.Broadcast(Reason);
 }
 
+bool USpatialGameInstance::GetPreventAutoConnectWithLocator() const
+{
+	// Need a more robust way of working out if we are the server
+	// World->IsServer() can not be used as on clients the NetDriver and DemoNetDriver on the world are both not setup yet and are null.
+	if (GetSpatialWorkerType() == SpatialConstants::DefaultServerWorkerType)
```

Comment:
Hmmm I believe this means that Offloaded workers will fail with this right?

Reply:
Why would that be?